### PR TITLE
[docs] Added zerotier setup tutorial

### PR DESCRIPTION
**Pull request by Aryamanz29**

### CODE REVIEW

File: user/network-topology.rst [hunk 1]
```diff
@@ -122,6 +122,101 @@ Sending data for topology with RECEIVE strategy
...
+   a topology receive URL is generated. Make sure you copy
+   that URL for later use in the topology script.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-network-topology/docs/docs/zerotier-tutorial/topology-2.png
+   :alt: ZeroTier topology configuration example 2
+
+2. Create a script for sending ZeroTier topology data
+#####################################################
+
+1. Now, create a script (e.g: ``/opt/send-zt-topology.sh``)
```
Comment (atb00ker) [inline user/network-topology.rst:165]: When following guides on OpenWRT related things, when I encounter something like "Create a script", the first question I have is where...
My laptop, the openwrt device or server.

Can you please mention that here?

File: user/zerotier.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+How to setup ZeroTier Tunnels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the procedure described below to setup
+ZeroTier tunnels on your devices.
+
+**Note:** This example uses **Shared systemwide (no organization)**
+option as the organization for VPN server and VPN client template.
+You can use any organization as long as VPN server, VPN client
+template and Device has same organization.
```
Comment (atb00ker) [inline user/zerotier.rst:20]: Not to be done in the this PR, but perhaps we can move this boilerplate to a file and link that file  in all the docs!

File: user/zerotier.rst [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+1. Configure Self-Hosted ZeroTier Network Controller
+####################################################
+
+If you haven't already set up a self-hosted Zerotier
```
Comment (atb00ker) [inline user/zerotier.rst:25]: Can we please mention here that we don't support the Zerotier SAAS offering yet and you *need* the self-hosted version.

Comment (Aryamanz29) [reply user/zerotier.rst]: Sure!

File: user/zerotier.rst (hunk 2 above)
Comment (Aryamanz29) [reply user/zerotier.rst:20]: I'll create issue for that, thanks!

File: user/network-topology.rst (hunk 1 above)
Comment (Aryamanz29) [reply user/network-topology.rst]: Sure!

**Review by pandafy — APPROVED** [review]
LGTM! 👍🏼

**Review by nemesifier — APPROVED** [review]

**Pull request merged**